File: [프로그래머스] 연습문제_이상한 문자 만들기.py
def solution(s):
    word = s.split(" ")
    print(word)
    answer = []
    for i in range(len(word)):
        st = list(word[i])
        for j in range(len(st)):
            if j%2 == 0:
                answer.append(st[j].upper())
            else:
                answer.append(st[j].lower())
        answer.append(' ')
    return "".join(answer[:-1])

File: [프로그래머스] 연습문제_이상한 문자 만들기.js
function solution(s) {
    var answer = '';
    s.split(' ').forEach((item)=>{
        for(let i=0; i < item.length; i++){
            i%2 === 0 ? answer += item[i].toUpperCase(): answer += item[i].toLowerCase();
        }
        answer += ' '
    })
    return answer.slice(0, -1);
}